### PR TITLE
Fix installation script

### PR DESCRIPTION
**Pull request by maturar**
Issues: #208 

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [x] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -22,11 +22,10 @@
     "rust-analyzer.cargo.unsetTest": [
         "api",
         "common",
-        "agent",
-        "server",
+        "ank-agent",
+        "ank-server",
```
Comment (inf17101) [inline .vscode/settings.json:26]: Why is it renamed, I thought it must match the paket name?

Comment (maturar) [reply .vscode/settings.json:26]: It must match to the name configured in the `Cargo.toml`.

Comment (inf17101) [reply .vscode/settings.json:26]: Ok 👍

**Review by inf17101 — APPROVED** [review]
LGTM 👍

**Pull request merged**